examples/physicalmachine-example.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 Chaos Mesh Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: chaos-mesh.org/v1alpha1
+kind: PhysicalMachine
+metadata:
+  name: physical-machine-example
+  labels:
+    region: region-a
+    arch: amd64
+spec:
+  # set the address of chaosd server
+  address: http://123.123.123.123:2333
```

Comment:
It's best to set the address with the `chaosd` server default port, not the dashboard default port. This will give meaning to the example.